generator/vogtle_data_generator.py

```diff
@@ -239,23 +229,44 @@ def generate_calendar(self, start_year, end_year):
                 output_file.write("%s|%s\n" % (i, day))
 
 
-if __name__ == '__main__':
+def generate_default(args=None, config=None):
+    """Generate the default set of synthetic data
+    """
     logging.info("Generating synthetic data...")
-    if not sys.argv[1:]:
-        ARG_PARSER.print_help()
-        ARG_PARSER.exit()
-
-    OPTIONS = {}
+    options = {}
     try:
-        OPTIONS = ARG_PARSER.parse_args()
-    except ArgumentError as err:
-        ARG_PARSER.print_help()
+        if args:
+            options = ARG_PARSER.parse_args(args)
```

Comment:
If you are doing a lot of command line function the package [Click](https://click.palletsprojects.com/en/7.x/) is pretty nice.